templates/index.json-ld

```diff
@@ -96,6 +96,18 @@
 				"language": "fr",
 				"path": "service-fr.html"
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility assessment #1 - GC theme template",
+				"language": "en",
+				"path": "reports/a11y-1-en.html"
+			},
+			{
+				"title": "Évaluation d'accessibilité #1 - Modèle de thème GC",
+				"language": "fr",
+				"path": "reports/a11y-1-fr.html"
+			}
```

Comment:
TODO: Move this need into the "index.json-ld" of the theme page once PR #2390 is merged.

Note: The broken link will be fixed after that move.

Reply:
Fixed in recent commit